en/06_Project_Governance/06_Minor_release_policy.md

```diff
@@ -0,0 +1,102 @@
+
+## Scope of the policy
+
+This policy applies to all [Silverstripe CMS commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the `silverstripe` github organisation that are not commercially supported are updated on a best effort basis.
+
+## General approach to major releases
+
+This policy complement the [Silverstripe CMS Major release policy](major_release_policy) by providing medium term guidance on when new minor releases of Silverstripe CMS are shipped.
+
+Note that the release cadence and pre-release time frame are indicative only. We may alter those in response to external constraints.
+
+Our minor release policy is more flexible because minor release upgrades are more straight forward to manage than major release upgrades. 
+
+## Upgrading to a new major release
+
+Silverstripe CMS follows [semantic versioning](https://semver.org/). Silverstripe CMS minor releases deliver new features and new APIs in a backward compatible way.
+
+## Minor release cadence
+
+Silverstripe CMS aims to ship a new minor releases every 6 months for the major release line in active development. Those minor releases are targeted for April and October each year.
+
+### Pre-release
+
+Silverstripe CMS does not usually tag alpha releases for minor releases. Changes intended to be shipped in the next minor release are tracked in *minor release branches* for each module. Those branches are normally named to an integer value. For example, the next minor release branch of `silverstripe/installer` 5 is called `5` and can be installed with `composer create-project silverstripe/installer:5.x-dev mysite`.
+
+Approximatively 6 weeks prior to the anticipated stable minor release, a beta minor release is published. Once a beta release is tagged, any new feature or API that didn't make it to the beta should be targeted to the follow up minor release. This allows the CMS development team to perform a regression test on the minor release. 
+
+The next minor release branch of each module is forked to a *patch release branch*. Patch release branches normally follow the pattern `MAJOR.MINOR`. For example, the patch release branch for `silverstripe/installer` 5.1 is called `5.1` and can be installed with `composer create-project silverstripe/installer:5.1.x-dev mysite`.
+
+Approximatively 2 weeks prior to the anticipated stable minor release, a release candidate is tagged. The release candidate is sent to an external auditor for a security review. The auditor normally received confidential security patches as well for review.
+
+### Stable release
+
+Once the team is ready, stable versions of all supported modules are tagged. A complete changelog is posted to the Silverstripe CMS documentation site along with any relevant upgrade notes.
+
+## Minor release support timeline
+
+Minor release support timeline follow similar phases to major release. But those phases are far more condensed.
```

Comment:
"those are farm more condensed" is ambiguous - is it the major or the minor timeline that is more condensed?
```suggestion
The minor release support timeline follows similar phases to those of a major release, but are more condensed.
```

Reply:
Suggestion applied.